resolver-dns/src/test/java/io/netty/resolver/dns/SearchDomainTest.java

```diff
@@ -33,20 +33,19 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.StringContains.containsString;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.core.StringContains.containsString;
 
 public class SearchDomainTest {
 
     private DnsNameResolverBuilder newResolver() {
         return new DnsNameResolverBuilder(group.next())
             .channelType(NioDatagramChannel.class)
-            .nameServerAddresses(DnsServerAddresses.singleton(dnsServer.localAddress()))
```

Comment:
@Scottmitch I wonder if we should maybe keep the method, mark it as `@deprecated` and just delegate to `nameServerProvider(...)`. This will allow more easily to upgrade for people. I know its marked as unstable but maybe its worth it. WDYT ? Also /cc @trustin